.gitignore

```diff
@@ -96,3 +96,22 @@ gradle-app.setting
 
 # Production data
 /.docker/
+
+
+### React
+# dependencies
+**/node_modules
+
+# testing
+**/coverage
+
+## Remove build
+**/client/build
+**/main/resources/static
```

Comment:
`**/main/resources/static` does not have to be excluded I think since we copy direct into the `build/` dir instead of the `src/`.